draft-pda-protocol.md

```diff
@@ -801,43 +804,50 @@ which has the following structure:
 
 ~~~
 struct {
-  Proto proto;
-  select (OutputShare.proto) {
+  select (Param.proto) { // Param corresponding to CollectReq.task_id
     case prio: PrioOutputShare;
     case hits: HitsOutputShare;
   }
 } OutputShare;
 ~~~
 
-Next, the helper encrypts its output share under the collector's HPKE public key:
+Next, the helper encrypts its output share under the collector's HPKE public
+key:
 
 ~~~
-enc, context = SetupBaseS(pk, "pda output share")
-encrypted_output_share = context.Seal(output_share,
-                            task_id || batch_start || batch_end)
+enc, context = SetupBaseS(pk, "pda output share" || server_role)
+encrypted_output_share = context.Seal(task_id || batch_start || batch_end,
+                                      output_share)
```

Comment:
Besides fixing the order of the arguments to `context.Seal`, I also appended `server_role` to the `info` string passed to `SetupBaseS`, since both helper and leader now encrypt output shares.